packages/shared/providers.yaml

```diff
@@ -755,11 +755,15 @@ buildium:
             type: string
             title: Client ID
             description: Your API Key Client ID
+            pattern: '^[a-z0-9_-]+$'
+            example: a228f0e7-b4a3-4150-b9ae-8552fc2880d3
     credentials:
         apiKey:
             type: string
             title: API Key
             description: The API key for your Buildium account
+            pattern: '^[a-z0-9_-]+$'
+            example: uOq2p+xlgpFdijfV/HqY+EvYpZKHRwlyhGuAVbJIxXs
```

Comment:
the regex doesn't match the value. It contains `+` or `/` for instance. You probably want `^[A-Za-z0-9+/]+$` instead

Reply:
aah yes, updated with [commit](https://github.com/NangoHQ/nango/pull/2818/commits/457afb07627c43917bf969a1cdaca95a6081af97)